README.md

```diff
@@ -2,6 +2,8 @@
 
 dokku-docker-auto-volumes is a plugin for [dokku][dokku] that automatically persist docker volumes declared in the Dockerfile.
 
+/var/lib/dokku/volumes 
```

Comment:
I think this line should replace the one at line 24, no? And with `$APP` at the end :)